### PR TITLE
JWWEB-735: Add assets list for home page

### PR DESCRIPTION
**Pull request by stcherenkov**
Requires #776 and some integration work

### CODE REVIEW

**Comment by stcherenkov** [comment]
Nicely done, thanks!

File: src/utils/address/startsWithOrEndsWith.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// @flow strict
+
+import { endsWith } from 'lodash-es'
+
+export function startsWithOrEndsWith(initialAddress: string, initialQuery: string): boolean {
+  const address = initialAddress.toLowerCase()
+  const query = initialQuery.toLowerCase()
+
+  return address.startsWith(query)
+    || address.startsWith(`0x${query}`)
```
Comment (IViol) [inline src/utils/address/startsWithOrEndsWith.js]: isn't it possible to pass query, that already contain `0x`?
probably, need to use `add0x` util function here

Comment (Creepypoke) [reply src/utils/address/startsWithOrEndsWith.js]: `.startsWith` [is not works](http://kangax.github.io/compat-table/es6/#test-String.prototype.startsWith) in Internet Explorer

File: src/pages/Home/HomeView.js [hunk 2]
```diff
@@ -1,64 +1,157 @@
...
+        return (
+          <li key={address}>
+            <AssetItem address={address} />
+          </li>
+        )
+      })}
+    </ul>
+  )
+
+  renderEmptyList = () => (
```
Comment (stcherenkov) [inline src/pages/Home/HomeView.js]: Separate component maybe?

File: src/utils/address/startsWithOrEndsWith.js (hunk 1 above)
Comment (stcherenkov) [reply src/utils/address/startsWithOrEndsWith.js]: Does babel transform it for IE?

File: src/pages/Home/HomeView.js (hunk 2 above)
Comment (Creepypoke) [reply src/pages/Home/HomeView.js]: yes, we make it in the next iteration

**Comment by stcherenkov** [comment]
👍

**Pull request merged**